src/components/ebay-carousel/index.js

```diff
@@ -40,9 +40,10 @@ function getInitialState(input) {
         items.length -= items.length % itemsPerSlide;
         // Only allow autoplay option for discrete carousels.
         if (input.autoplay) {
+            const isSingleSlide = items.length <= itemsPerSlide;
             state.autoplayInterval = parseInt(input.autoplay, 10) || 4000;
             state.classes.push('carousel__autoplay');
-            state.paused = input.paused;
+            state.paused = isSingleSlide || input.paused; // Force paused state if not enough slides provided;
```

Comment:
shouldn't this be 'isSingleSlide && input.paused;' instead of 'isSingleSlide || input.paused;' ?